src/main/res/values/attrs.xml

```diff
@@ -5,6 +5,7 @@
     </declare-styleable>
 
     <declare-styleable name="CountryChooser">
+        <attr name="defaultSelectionText" format="string" />
```

Comment:
Also, we should generate new languages for countries list:
English, Russian, Korean, Japanese, Spanish and Turkish